BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -121,57 +96,57 @@ public struct AccessibilityElement: Element {
 
 extension Element {
```

Comment:
The changes here look complicated, but it's really just switching the order in which these functions are defined—the deprecated function is now below the new one.